docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+volumes:
+  db:
+
+services:
+
+  db:
+    image: mysql
+    env_file: .env
+
+    volumes:
+      - db:/var/lib/mysql
+
+  web:
+    build: .
+    image: ualbertalib/dmp_roadmap
+    env_file: .env
+    command: bash -c 'rm -f tmp/pids/server.pid && rake db:migrate && rails s -p 3000 -b "0.0.0.0"'
```

Comment:
have rake db:migrate here will not work for the first time, rather, we could have a script like this to run the series of command needed:

```
docker-compose up -d
docker-compose run --rm web rake db:migrate
docker-compose run --rm web rake assets:precompile
```